index.bs

```diff
@@ -3406,6 +3421,68 @@ and |session|:
 
 </div>
 
+<div algorithm="collect nodes using accessibility attributes">
+To <dfn>collect nodes using accessibility attributes</dfn> with given |context nodes|, |selector|,
+|maximum returned node count|, and |returned nodes|:
+
+1. If |returned nodes| is null:
+
+  1. Set |returned nodes| to an empty [=/list=].
+
+1. For each |context node| in |context nodes|:
+
+  1. Let |match| be true.
+
+  1. If |selector|'s <code>role</code> is present:
+
+    1. Let |role| be the [=computed role=] of |context node|.
+
+    1. If |selector|'s <code>role</code> [=is|is not=] |role|:
```

Comment:
Not sure how complex the computation of both the role and name are, but we won't have to do it when the given key hasn't been specified, right?

Reply:
yes, I believe the key is checked above at line 3436